retriever/lib/repository.py

```diff
@@ -9,25 +11,19 @@
 import urllib.error
 import imp
 from pkg_resources import parse_version
-from retriever import REPOSITORY, SCRIPT_WRITE_PATH, HOME_DIR
+from retriever.lib.defaults import REPOSITORY, SCRIPT_WRITE_PATH, HOME_DIR
 from retriever.lib.models import file_exists
 
-global abort, executable_name
-abort = False
-executable_name = "retriever"
 
-
-def download_from_repository(filepath, newpath, repo=REPOSITORY):
-    """Downloads the latest version of a file from the repository."""
+def _download_from_repository(filepath, newpath, repo=REPOSITORY):
+    """Download latest version of a file from the repository."""
     try:
-        filename = filepath.split('/')[-1]
         urllib.request.urlretrieve(repo + filepath, newpath)
     except:
         raise
-        pass
 
 
-def check_for_updates():
+def check_for_updates(quite=True):
```

Comment:
quite -> quiet

Reply:
If we're defaulting to `quiet=False` for installs is there a reason to default to `quiet=True` here?

Reply:
Yes, this does seem inconsistent. Will change `quiet=True`. On another note the `quiet` param has been removed from `install` functions.

Reply:
Thanks. Ping this issue when you're done with any further changes and we'll work on getting this in.

Reply:
@ethanwhite , This is covered